src/ATen/DLConvertor.cpp

```diff
@@ -13,28 +13,28 @@ static DLDataType getDLDataType(const Type& type) {
   dtype.bits = type.elementSizeInBytes() * 8;
   switch (type.scalarType()) {
     case ScalarType::Byte:
-      dtype.code = DLDataTypeCode::kUInt;
+      dtype.code = DLDataTypeCode::kDlUInt;
```

Comment:
I think we should use `DL` because that is the namespace prefix being used elsewhere.